gax-java/README.md

```diff
@@ -15,7 +15,7 @@ files can use services such as paged list iteration, request batching, and
 polling of long-running operations to provide a more convenient and idiomatic
 API surface to callers.
 
-Currently, this library shouldn't be used independently from google-cloud-java, otherwise there is
+Currently, this library shouldn't be used independently of google-cloud-java, otherwise there is
 a high risk of diamond dependency problems, because google-cloud-java uses beta features from this
 library which can change in breaking ways between versions. See [VERSIONING](#versioning) for
 more information.
```

Comment:
Noticed that some other content of this readme need to be updated too. e.g. The  version below is still `2.8.1`, the `Build Status` above is not working anymore. It could be a separate PR though.

Reply:
Updated to the latest version and removed the build status badge.